features/backend/media.feature

```diff
@@ -0,0 +1,180 @@
+@backend
+Feature: Media settings
+    In order to update media settings
+    As an admin
+    I should be able to edit media settings
+
+Background:
+    Given following "Event":
+        | ref    | title          | description                | startDate        | endDate          | venue              | email               | host                      |
+        | event  | My event       | My another awesome event!  | 2016-03-01 10:00 | 2016-03-01 18:00 | Burj Khalifa Tower | eventator@email.com | http://localhost:8000     |
+        | event2 | My other event | My other awesome event!    | 2016-03-01 10:00 | 2016-03-01 18:00 | Burj Khalifa Tower | eventator@gmail.com | http://eventator.loc      |
```

Comment:
Please check https://github.com/pilot/eventator/blob/master/var/ci/eventator.loc.conf

In this config u can see port 8080, not 80! 😃  But in the event2 host value i see 80 port (!) That's the main reason why your test is fail.